osu.Game/Overlays/Mods/ModSelectColumn.cs

```diff
@@ -73,6 +80,16 @@ protected ModSelectColumn()
                                     RelativeSizeAxes = Axes.X,
                                     Height = header_height + ModSelectPanel.CORNER_RADIUS
                                 },
+                                triangles = new TrianglesV2
+                                {
+                                    Anchor = Anchor.TopRight,
+                                    Origin = Anchor.TopRight,
+                                    RelativeSizeAxes = Axes.X,
+                                    Width = 1.1f,
```

Comment:
This `1.1f` is not correct. It'll look okay because the excess is masked, and maybe that's fine, but this needs to be mentioned in an inline comment. Alternatively, as the width and height is already known a more correct value could be specified.

If you need a method of figuring out what is correct, you can use a solid colour box to visualise the area which is being drawn:

![JetBrains Rider-EAP 2023-02-17 at 04 02 16](https://user-images.githubusercontent.com/191335/219546459-6e9b41e1-fc8b-451f-9cb1-7bc2cd96ee4a.png)

Note that `1.03f` is already enough to cover the area.

Reply:
Note that I wasn't suggest just using `1.03f` 😅

Maybe it's okay but still a bit suspicious.

Reply:
Came up with better approach.
We can remove masking from triangles since it will be masked by top-level containers. However doing so will result in some pixels visible in unwanted areas [(see screenshot)](https://user-images.githubusercontent.com/22874522/206163751-c8ff41dd-9118-40cc-8b16-ca580c63561c.png). What if we could make bottom area transparent, so these pixels could disappear.. And we can! Since we are applying gradient to the triangles - we can just specify bottom colour as having 0 alpha.
Commit should be self-explanatory.